components/property_info.rst

```diff
@@ -1,26 +1,446 @@
 .. index::
-   single: PropertyInfo
-   single: Components; PropertyInfo
+    single: PropertyInfo
+    single: Components; PropertyInfo
 
 The PropertyInfo Component
 ==========================
 
-    The PropertyInfo component extracts information about PHP class properties
-    using metadata from popular sources, like `Reflection`_ and `PHP Documentation`_.
+    The PropertyInfo component provides the functionality to get information
+    about class properties using metadata of popular sources.
+
+While the :doc:`PropertyAccess component </components/property_access/introduction>`
```

Comment:
The path should be updated to `/components/property_info` (it's why Travis fails)